docs/index.html

```diff
@@ -83,7 +83,7 @@ <h2 class="title">A 1x Engineer...</h2>
           <li>Sometimes loses their work.</li>
           <li>Doesn't have to have the entire codebase memorized.</li>
           <li>Respects and upholds community Codes of Conduct.</li>
-          <li>May work from home, the office, a coffee shop, or where ever else best works for them.</li>
+          <li>May work from home, the office, a coffee shop, or wherever else best works for them.</li>
```

Comment:
where ever is actually preferred here, in the way that it is used emphatically

Reply:
Thanks! The latest commit reverts this change.